wagl/brdf.py

```diff
@@ -552,77 +416,34 @@ def find_file(files, brdf_wl, parameter):
         brdf_base_dir = brdf_primary_path
         brdf_dirs = get_brdf_dirs_modis(brdf_base_dir, dt)
 
-    # The following hdflist code was resurrected from the old SVN repo. JS
     # get all HDF files in the input dir
     dbDir = pjoin(brdf_base_dir, brdf_dirs)
-    three_tup = os.walk(dbDir)
-    hdflist = []
-    hdfhome = None
+    tile_list = [pjoin(folder, f)
+                 for (folder, _, filelist) in os.walk(dbDir) for f in filelist if f.endswith(".h5")]
 
-    for (hdfhome, _, filelist) in three_tup:
-        for f in filelist:
-            if f.endswith(".hdf.gz") or f.endswith(".hdf"):
-                hdflist.append(f)
+    src_poly, src_crs = valid_region(acquisition.uri, acquisition.no_data)
+    src_crs = rasterio.crs.CRS(**src_crs)
 
-    results = {}
-    for param in BrdfParameters:
-        hdf_fname = find_file(hdflist, acquisition.brdf_wavelength,
-                              param.name.lower())
+    tally = {}
 
-        hdfFile = pjoin(hdfhome, hdf_fname)
+    for ds in acquisition.brdf_datasets:
+        tally[ds] = BrdfTileSummary.empty(use_JuppLi_brdf)
+        for tile in tile_list:
+            with h5py.File(tile, 'r') as fid:
+                tally[ds] += load_brdf_tile(src_poly, src_crs, fid, ds, use_JuppLi_brdf)
```

Comment:
Just a minor comment, that I would like addressed.
Instead of the variable *use_JuppLi_brdf*, I'd like to a little more explicit about what it is and be consistent.  So I'd suggest a variable name like *fallback_brdf* or similar.  I would prefer to keep consistency when we're referring to fallback datasets.